src/NzbDrone.Common/Processes/PidFileProvider.cs

```diff
@@ -12,12 +12,14 @@ public interface IProvidePidFile
 
     public class PidFileProvider : IProvidePidFile
     {
+        private readonly IStartupContext _startupContext;
         private readonly IAppFolderInfo _appFolderInfo;
         private readonly IProcessProvider _processProvider;
         private readonly Logger _logger;
 
-        public PidFileProvider(IAppFolderInfo appFolderInfo, IProcessProvider processProvider, Logger logger)
+        public PidFileProvider(IStartupContext startupContext, IAppFolderInfo appFolderInfo, IProcessProvider processProvider, Logger logger)
```

Comment:
A bit pedantic, but the new class should be put at the end of the list (but before logger).

```
public PidFileProvider(IAppFolderInfo appFolderInfo, IProcessProvider processProvider, IStartupContext startupContext, Logger logger)
```